yarn.lock

```diff
@@ -13782,6 +13782,13 @@ ovh-api-services@^9.44.0:
   dependencies:
     lodash "^4.17.15"
 
+ovh-api-services@^9.43.0:
```

Comment:
Please refer to my previous comment related to `ovh-api-services`.

Rebasing on `develop` branch will remove this extra diff.

Reply:
Done